java/code/src/com/redhat/rhn/frontend/xmlrpc/user/external/UserExternalHandler.java

```diff
@@ -60,7 +60,7 @@
      *
      * @xmlrpc.doc Set whether we should keeps roles assigned to users because of
      * their IPA groups even after they log in through a non-IPA method. Can only be
-     * called by a satellite_admin.
+     * called by a #product() admin.
```

Comment:
`satellite_admin` is the role label in DB. The text version is `SUSE Manager Administrator`.
So maybe we write not "#product() admin" but "#product() Administrator" ?

Here and below.

Reply:
Thanks, I added another commit (6d4baa4) with those changes, turning also additional occurrences of `Sat admin` to `product admin` in javadoc.